source/lib-intro.tex

```diff
@@ -3309,6 +3309,13 @@
 operator delete[](void*, std::align_val_t, const std::nothrow_t&)
 \end{codeblock}
 
+\pnum
+A \Cpp{} program may provide the definition of
+the following function signature declared in header \libheaderref{debugging}:
+\begin{codeblock}
+std::is_debugger_present()
```

Comment:
I think we were missing the full declaration here (return type and noexcept), fixing.